G/GDAL/build_tarballs.jl

```diff
@@ -0,0 +1,74 @@
+using BinaryBuilder
+
+name = "GDAL"
+version = v"3.0.2"
+
+# Collection of sources required to build GDAL
+sources = [
+    "https://github.com/OSGeo/gdal/releases/download/v$version/gdal-$version.tar.gz" =>
+    "787cf150346e58bff0ccf8c131b333139273e35d2abd590ad7196a9ee08f0039",
+]
+
+# Bash recipe for building across all platforms
+script = raw"""
+cd $WORKSPACE/srcdir/gdal-*/
+
+if [[ ${target} == *mingw* ]]; then
+    export LDFLAGS="${libdir}"
```

Comment:
My fault, I missed a `-L` in my suggestion:
```suggestion
    export LDFLAGS="-L${libdir}"
```
:grimacing:

Reply:
Haha, and I'm clueless enough to miss that. Better this than having to patch GDAL :)